<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hamming Distance Claculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter First String:</string>
    </property>
    <widget class="QLineEdit" name="lineEditStr1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Second String:</string>
    </property>
    <widget class="QLineEdit" name="lineEditStr2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>311</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResult">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClear">
    <property name="title">
     <string>Clear</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuClear"/>
  </widget>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
